src/media/decoding.rs

```diff
@@ -49,41 +49,7 @@ impl<R: Read + Send + Sync> symphonia::core::io::MediaSource for MediaInput<R> {
 
 /// Media stream decoder (OGG, WAV, FLAC, ..)
 ///
-/// Using the `MediaDecoder` is the preferred way to play large audio files and streams. For small
-/// soundbites, consider using
-/// [`decode_audio_data_sync`](crate::context::BaseAudioContext::decode_audio_data_sync) on the
-/// audio context which will create a single AudioBuffer which can be played/looped with high
-/// precision in an `AudioBufferSourceNode`.
-///
-/// The MediaDecoder implements the [`MediaStream`](crate::media::MediaStream) trait so can be used
-/// inside a `MediaStreamAudioSourceNode`. Please note that this means the decoding will take place
-/// on the render thread which is typically not desired. In a later version of this library, we
-/// will add a buffered version which will decode in a separate thread.
-/// <https://github.com/orottier/web-audio-api-rs/issues/120>
-///
 /// The current implementation can decode FLAC, Opus, PCM, Vorbis, and Wav.
-///
-/// # Warning
-///
-/// This abstraction is not part of the Web Audio API, it is only provided for
-/// convenience reasons.
-///
-/// # Example
-///
-/// ```no_run
-/// use web_audio_api::context::{AudioContext, BaseAudioContext};
-/// use web_audio_api::media::MediaDecoder;
-/// use web_audio_api::node::AudioNode;
-///
-/// // build a decoded audio stream the decoder
-/// let file = std::fs::File::open("samples/major-scale.ogg").unwrap();
-/// let stream = MediaDecoder::try_new(file).unwrap();
-/// // pipe the media stream into the web audio graph
-/// let context = AudioContext::default();
-/// let node = context.create_media_stream_source(stream);
-/// node.connect(&context.destination());
-/// ```
-///
 pub struct MediaDecoder {
```

Comment:
I saw this commit message: "Remove MediaDecoder from public API", shouldn't it be `pub(crate)` then ?

Reply:
The mod is private so this is not strictly necessary.  I will add it for clarity though!